BetonQuest-core/src/main/java/pl/betoncraft/betonquest/commands/QuestCommand.java

```diff
@@ -17,6 +17,7 @@
  */
 package pl.betoncraft.betonquest.commands;
 
+import de.slikey.effectlib.effect.LoveEffect;
```

Comment:
Any reason for this import?

Reply:
you are right, i don't know what happened